listenbrainz/listenstore/influx_listenstore.py

```diff
@@ -347,6 +347,73 @@ def fetch_listens_from_storage(self, user_name, from_ts, to_ts, limit, order):
 
         return listens
 
+    def get_listens_batch_for_dump(self, username, dump_time, offset):
```

Comment:
I'm not convinced that this is the fastest way to do this -- but I think it is good enough for now. Like my other comment, we don't know what is going to stick just yet.